process_avocado_results.py

```diff
@@ -37,6 +37,7 @@ def sanitize(testname):
     """
     testname = testname.split('-', 1)[-1]    # Strip off leading 'XX-'
     testname = testname.replace('.py:', '.') # Remove filename suffixes
+    testname = testname.rsplit(';run-', 1)[0] # Strip off trailing ';run-XXXX'
```

Comment:
I'm wondering whether you should be using https://github.com/avocado-framework/avocado/tree/master/optional_plugins/resultsdb (with some additions) instead of cooking a custom wrapper. Feel free to reach us for details...